utils/files.ts

```diff
@@ -40,14 +59,15 @@ export const formatPaybuttonTransactionsFileData = (data: TransactionFileData):
     date,
     value,
     rate,
-    currency
+    newtworkId
   } = data
+  const networkTicker = NETWORK_TICKERS_FROM_ID[newtworkId]
 
   return {
     ...data,
-    amount: amount.toFixed(DECIMALS[currency]),
+    amount: amount.toFixed(DECIMALS[networkTicker]),
     date: date.format(PRICE_API_DATE_FORMAT),
-    value: value.toFixed(2),
+    value: value.toFixed(DECIMALS[networkTicker]),
```

Comment:
This is wrong, will lead to USD/CAD values having 8 decimal digits if the networkId is 2, for BCH.

Should be `DECIMALS.FIAT`.